vector/src/main/java/im/vector/home/BaseNewHomeIndividualFragment.kt

```diff
@@ -0,0 +1,90 @@
+package im.vector.home
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.View.GONE
+import androidx.core.content.ContextCompat
+import im.vector.R
+import im.vector.adapters.AbsAdapter
+import im.vector.adapters.HomeRoomAdapter
+import im.vector.fragments.AbsHomeFragment
+import im.vector.ui.themes.ThemeUtils
+import kotlinx.android.synthetic.main.fragment_home_individual.*
+import org.matrix.androidsdk.data.Room
+import java.util.*
+import kotlin.collections.ArrayList
+
+
+abstract class BaseNewHomeIndividualFragment : AbsHomeFragment(), UpDateListener {
```

Comment:
"NewHome" isn't very descriptive, what is new about this class?  Code comments would help with the understanding too,

Reply:
renamed to LingoHome